test/uk/gov/hmrc/gform/gform/handlers/DeclarationControllerRequestHandlerSpec.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.gform.handlers
+
+import play.api.libs.json.Json
+import uk.gov.hmrc.gform.Spec
+import uk.gov.hmrc.gform.sharedmodel.ExampleData
+import uk.gov.hmrc.gform.sharedmodel.form.{ Form, FormField }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormComponentId
+
+class DeclarationControllerRequestHandlerSpec extends Spec {
+
+  it should "handle UpdatedFormRequest" in new DeclarationControllerRequestHandler {
+
+    val actual: Option[Form] =
+      handleUpdatedFormRequest(Option(requestBody), ExampleData.form)
+
+    actual.get.formData.fields should contain(newFormField)
```

Comment:
You should use `.value` from instead of `.get`. Difference is in stacktrace:

With `.get`:
```
[info] DeclarationControllerRequestHandlerSpec:
[info] - should handle UpdatedFormRequest *** FAILED ***
[info]   java.util.NoSuchElementException: None.get
[info]   at scala.None$.get(Option.scala:347)
[info]   at scala.None$.get(Option.scala:345)
[info]   at uk.gov.hmrc.gform.gform.handlers.DeclarationControllerRequestHandlerSpec$$anonfun$1$$anon$1.<init>(DeclarationControllerRequestHandlerSpec.scala:32)
[info]   at uk.gov.hmrc.gform.gform.handlers.DeclarationControllerRequestHandlerSpec$$anonfun$1.apply(DeclarationControllerRequestHandlerSpec.scala:27)
[info]   at uk.gov.hmrc.gform.gform.handlers.DeclarationControllerRequestHandlerSpec$$anonfun$1.apply(DeclarationControllerRequestHandlerSpec.scala:27)
[info]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.FlatSpecLike$$anon$1.apply(FlatSpecLike.scala:1682)
[info]   ...
```

With `.value`
```
[info] DeclarationControllerRequestHandlerSpec:
[info] - should handle UpdatedFormRequest *** FAILED ***
[info]   The Option on which value was invoked was not defined. (DeclarationControllerRequestHandlerSpec.scala:32)
```

Btw. that `.value` is coming from `org.scalatest.OptionValues` and is already in scope.

Reply:
changed to .value, but this test needs improvement anyway I don't like .get and .value 
this tickets needs enhancement later on so I'll improve next week